morse_code.rb

```diff
@@ -0,0 +1,26 @@
+def decode_char(char)
+  morse_converter = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', ' ' => ' '
+  }
+  return morse_converter[char]
+end
+
+def decode_word(word)
+  word_split = word.split(' ')
+  array = word_split.map do |i|
+    decode_char(i)
+  end
+  return array.join
+end
+
+def decode_sentence(phrase)
+  phrase_split = phrase.split('   ')
+  array = phrase_split.map do |word|
+    decode_word(word)
+  end
+  return array.join(' ')
```

Comment:
- [ ] <b>OPTIONAL:</b> This function has a descriptive name and it is fine, but there was a little bit of confusion when reviewing since the project requires to call it just `decode`. So, to avoid such confusion in the future, It would be very nice to follow project instructions as there are on those simple details.

Reply:
Thanks  rukundoeric, You're totally right, We will apply this recommendation to our next projects 😄